src/components/TodoList/TodoList.jsx

```diff
@@ -1,28 +1,61 @@
 import React from 'react';
 import './TodoList.scss';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
+export const TodoList = ({ todos, userDeteil }) => (
   <div className="TodoList">
     <h2>Todos:</h2>
 
     <ul className="TodoList__list">
-      <li className="TodoList__item">
-        <label>
-          <input type="checkbox" readOnly />
-          delectus aut autem
-        </label>
 
-        <button type="button">User&nbsp;#1</button>
-      </li>
+      {todos.map(todo => (
+        <li
+          key={todo.id}
+          className={`TodoList__item ${todo.completed
+            && 'TodoList__item--completed'}`}
+        >
+          <label
+            className="TodoList__label"
+          >
+            <span className="TodoList__head">ID:</span>
+            <span className="TodoList__text">{todo.id}</span>
+            <input
+              className="TodoList__checkbox"
+              type="checkbox"
+              checked={todo.completed && true}
+              onChange={() => todo.completed}
+            />
+          </label>
 
-      <li className="TodoList__item">
-        <label>
-          <input type="checkbox" checked readOnly />
-          distinctio vitae autem nihil ut molestias quo
-        </label>
+          <div
+            className="TodoList__user"
+          >
+            <span className="TodoList__head">User ID:</span>
+            <span className="TodoList__text">{todo.userId}</span>
+          </div>
 
-        <button type="button">User&nbsp;#2</button>
-      </li>
+          <p
+            className="TodoList__title"
+          >
+            <span className="TodoList__head">Title:</span>
+            <span className="TodoList__text">{todo.title}</span>
+          </p>
+
+          <button
+            type="button"
+            className="TodoList__button"
+            value={+todo.userId}
+            onClick={userDeteil}
+          >
+            User&nbsp;#1
```

Comment:
this value should match the id of each user, but now each user has id # 1
![image](https://user-images.githubusercontent.com/26859185/93057451-9daa9f00-f676-11ea-9ce3-f337497723b1.png)